### PR TITLE
fix #7546 - styling for flat interface

### LINKED ISSUE #7546: Header bar and tab styling.
Header styling based on:

pasted image at 2017_02_20 02_39 pm

### PR DESCRIPTION
**Pull request by jamesmudgett**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Fixes https://github.com/brave/browser-laptop/issues/7546

Test Plan:

### CODE REVIEW

**Comment by bsclifton** [comment]
@jamesmudgett can you attach a screenshot? 😄

**Comment by jamesmudgett** [comment]
screen shot 2017-03-10 at 11 29 11 am

**Comment by jamesmudgett** [comment]
screen shot 2017-03-10 at 11 30 56 am

**Comment by bsclifton** [comment]
@bradleyrichter should this be configurable? Or should these changes replace what we have?

I really like it- great job @jamesmudgett 😄   I'd vote to have this replace what we have currently

**Comment by bradleyrichter** [comment]
@bsclifton It's almost ready. Just needs an adjustment for the BM bar and maybe some color adjustment. @jamesmudgett knocked it out while we were sitting in the all-hands.   :   )

**Comment by bradleyrichter** [comment]
and yes, it can replace what we have.

**Comment by luixxiul** [comment]
Testing the PR locally, I got the error on the console:

````
[83370:775:0311/223233.679360:ERROR:CONSOLE(0)] "(WEBUI context) http://localhost:8080/gen/app.entry.js:69936: Uncaught TypeError: Cannot read property 'getIn' of undefined{TypeError: Cannot read property 'getIn' of undefined
    at NavigationBar.get publisherId [as publisherId] (http://localhost:8080/gen/app.entry.js:69936:36)
    at NavigationBar.get visiblePublisher [as visiblePublisher] (http://localhost:8080/gen/app.entry.js:69944:53)
    at NavigationBar.get isPublisherButtonEnabled [as isPublisherButtonEnabled] (http://localhost:8080/gen/app.entry.js:69951:63)
    at NavigationBar.render (http://localhost:8080/gen/app.entry.js:70046:30)
    at http://localhost:8080/gen/app.entry.js:23765:22
    at measureLifeCyclePerf (http://localhost:8080/gen/app.entry.js:23044:13)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (http://localhost:8080/gen/app.entry.js:23764:26)
    at ReactCompositeComponentWrapper._renderValidatedComponent (http://localhost:8080/gen/app.entry.js:23791:33)
    at ReactCompositeComponentWrapper.performInitialMount (http://localhost:8080/gen/app.entry.js:23331:31)
    at ReactCompositeComponentWrapper.mountComponent (http://localhost:8080/gen/app.entry.js:23227:22)}", source: chrome://brave/Users/Suguru/browser-laptop/app/extensions/brave/index-dev.html (0)
[83370:775:0311/223233.724993:ERROR:CONSOLE(15985)] "Uncaught TypeError: Cannot read property '_idCounter' of null", source: http://localhost:8080/gen/app.entry.js (15985)
[83370:775:0311/223234.500201:ERROR:CONSOLE(15985)] "Uncaught TypeError: Cannot read property '_idCounter' of null", source: http://localhost:8080/gen/app.entry.js (15985)
[83370:775:0311/223242.675519:ERROR:CONSOLE(15985)] "Uncaught TypeError: Cannot read property '_idCounter' of null", source: http://localhost:8080/gen/app.entry.js (15985)
````

I've cleared ~/.electron and node_modules and done `npm i` before checking the PR.

**Comment by luixxiul** [comment]
But it looks to me that it is a different issue.

**Comment by luixxiul** [comment]
If you show the bookmark toolbar, the unnecessary border is displayed under the bookmark bar:

screenshot 2017-03-11 22 39 17

**Comment by bradleyrichter** [comment]
Yes. I tried removing both top and bottom border on the bm bar and it looks better.

**Comment by jamesmudgett** [comment]
I think removing the borders is a nice solution, cleaner and simpler.

**Comment by jamesmudgett** [comment]
screen shot 2017-03-11 at 12 40 59 pm

**Comment by bradleyrichter** [comment]
I also lightened the URL box border about 30% and tried a light grey to white gradient from top to bottom. (Clear bg on bm bar) This lightens up the chrome but adds contrast to the tab bar and still allows the white URL BG to show. See what you think...

**Comment by jamesmudgett** [comment]
@bradleyrichter can you post screenshot, or push to branch?

**Comment by bsclifton** [comment]
This is looking good! Here's a screenshot on macOS:
screen shot 2017-03-12 at 2 39 26 am

Notice now that after the borders have been removed, there is a color difference (dark block under bookmarks toolbar). That needs to have its color adjusted

**Comment by bsclifton** [comment]
Windows looks good too! 👍  Love these changes!! 😄 

Here's another thing I caught: notice there is no border under tabs which are not active. It looks a little weird.  In the following screenshot, notice the missing border under the Twitter tab
screen shot 2017-03-12 at 3 01 52 am

**Comment by jamesmudgett** [comment]
img_3235

Slight adjustment from original design tabs area intentionally slightly darker than bookmarks and address bar area to create a bit of contrast. See original design above instead of two tone for tabs area carried over darker gray into inactive tabs as well to boost contrast to active tab and simplify space. Good call on inactive tab bottom border.

**Comment by jamesmudgett** [comment]
screen shot 2017-03-13 at 5 33 07 pm

**Comment by cezaraugusto** [comment]
I did a rebase and after convo with @bradleyrichter on Slack this looks ready with only a minor touch-up but I think that can be done as a follow-up. 

I have some work to do on top of it so I'll merge. 

btw @jamesmudgett this looks awesome +++++++++

**Comment by Liunkae** [comment]
Just noticed the new look on master. Great stuff!

File: less/window.less [hunk 1]
```diff
@@ -39,13 +39,14 @@ html,
 }
 
 .top {
-    background: linear-gradient(to bottom, #fbfbfb, #ededed, );
+    background: linear-gradient(to bottom, #eaeaea, #f2f2f2, );
```
Comment (jonathansampson) [inline less/window.less]: Is that a stray `,` at the end of the gradient?

**Pull request merged**